<launch>
    <node name="snap_node" pkg="RYDS" type="snap_node.py" />
    <node name="compare_histo" pkg="RYDS" type="compare_histo.py" /> 
</launch>
